misc/testerina/modules/testerina-core/src/main/java/org/ballerinalang/testerina/natives/mock/ObjectMock.java

```diff
@@ -297,6 +303,10 @@ public static BError thenReturn(BObject caseObj) {
         return null;
     }
 
+    private static boolean validateFieldAccess(ObjectType objectType, String fieldName) {
```

Comment:
Shall we use a name like `validateFieldAccessIsPublic` or `isFieldPublic` for readability?

Reply:
Updated with 5b9bd8a